serverless/src/getConcepts/handler.js

```diff
@@ -33,6 +37,42 @@ import { toSkosJson } from '@/shared/toSkosJson'
  */
 export const getConcepts = async (event) => {
   const { defaultResponseHeaders } = getApplicationConfig()
+  const { queryStringParameters = {} } = event
+  const { format = '', scheme = '' } = queryStringParameters
+  if (format === 'csv') {
```

Comment:
will this work with concept scheme and pattern, if not, we should throw an error if user attempts to include those parameters

Reply:
that should work

Reply:
I don't think it will work, you only accept "scheme" as a parameter to your csv output.   So I think if they try and include pattern we should thrown an error or something.   Might be able to figure out how to generate csv with pattern but that isn't MVP for this ticket.

Reply:
Done